Standards/scs-0001-v1-sovereign-cloud-standards.md

```diff
@@ -83,7 +83,7 @@ embedded in the markdown header.
 | Field name | Requirement | Description |
 | --- | --- | --- |
 | `type` | REQUIRED | one of `Procedural`, `Standard`, or `Decision Record` |
-| `status` | REQUIRED | one of `Draft`, `Stable`, `Deprecated`, or `Rejected` |
+| `status` | REQUIRED | one of `Proposal`, `Draft`, `Stable`, `Deprecated`, or `Rejected` |
```

Comment:
I believe that the status `Proposal` that you add here already exists and is named `Draft`.
The subsequent description of the life cycle of an ADR or a Standard does NOT talk about a `Proposal` status, so adding it to the table actually makes the document less consistent.
If we think that we need another phase, we can add it of course, but then we should do so consistently. Currently we have Draft(pre-merge) -> Draft(merged) -> Stable -> Deprecated as lifecycle for accepted Documents, otherwise Draft(pre-merge) -> Draft(merged) -> Rejected

Reply:
I have to disagree. The ADR SCS-0001 describes the following process:

```mermaid
graph TD
    A[Proposal] -->|Pull Request| B[Draft]
    B -->|Pull Request| D[Stable]
    B -->|Pull Request| E[Rejected]
    D -->|Pull Request| F[Deprecated]
```

See [SCS-0001#Process](https://github.com/SovereignCloudStack/Docs/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md#process) for reference.

Reply:
https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md is the working link.
The text talks about `Proposal` at one place but in most places states that the doc starts out in the `Draft` state. That's why I only had a pre-merge Draft status in my mind when reading the description.
So we should either
(a) remove the `Proposal` from the document and replace it with pre-merge Draft OR
(b) use `Proposal` at all places as the first stage of the life-cycle
In any case, the above graph should be included (with adjusted wording in case (a)).

Reply:
We need to decide which way to go.
@horazont - care to weigh in?

Reply:
> (b) use `Proposal` at all places as the first stage of the life-cycle In any case, the above graph should be included (with adjusted wording in case (a)).

+1  for b) and adding the graph as it helps massively IMHO